sycl/plugins/unified_runtime/ur/adapters/level_zero/common.hpp

```diff
@@ -309,14 +298,22 @@ ur_result_t ze2urResult(ze_result_t ZeResult);
 // Trace a call to Level-Zero RT
 #define ZE2UR_CALL(ZeName, ZeArgs)                                             \
   {                                                                            \
-    ze_result_t ZeResult = ZeName ZeArgs;                                      \
-    if (auto Result = ZeCall().doCall(ZeResult, #ZeName, #ZeArgs, true))       \
+    urPrint("ZE ---> %s%s\n", #ZeName, #ZeArgs);                               \
+    if (auto Result = ZeCall().doCall(ZeName ZeArgs, #ZeName, #ZeArgs))        \
       return ze2urResult(Result);                                              \
   }
 
-// Perform traced call to L0 without checking for errors
+#define ZE_CALL(ZeName, ZeArgs, ZeResult)                                      \
+  {                                                                            \
+    urPrint("ZE ---> %s%s\n", #ZeName, #ZeArgs);                               \
+    ZeResult = ZeCall().doCall(ZeName ZeArgs, #ZeName, #ZeArgs);               \
+  }
```

Comment:
ZE_CALL not doing check now? Why is it OK?

Reply:
this ZE_CALL is the same as previous ZE_CALL_NOCHECK, just that it has the urPrint before, so it is keeping the behavior of not checking.

Reply:
then perhaps keep its name ZE_CALL_NOCHECK?